app/react/package.json

```diff
@@ -57,6 +57,7 @@
     "@storybook/react-docgen-typescript-plugin": "1.0.2-canary.6.9d540b91e815f8fc2f8829189deb00553559ff63.0",
     "@storybook/semver": "^7.3.2",
     "@storybook/store": "6.5.0-alpha.1",
+    "@types/node": "^14.14.20",
```

Comment:
`^14.14.20 || ^16.0.0`? And move `@types/node` to dependencies for other packages too?

See https://github.com/storybookjs/storybook/pull/16894/files

Reply:
I've added it for the other relevant packages. I'll keep it as 14 unless we somehow combine these PRs.